SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -866,20 +861,19 @@ void CNSSolver::SetTauWall_WF(CGeometry *geometry, CSolver **solver_container, c
        iteratively solve for a new wall shear stress. Use the current wall
        shear stress as a starting guess for the wall function. ---*/
 
-      unsigned long counter = 0; su2double diff = 1.0;
-      U_Tau = sqrt(WallShearStress/Density_Wall);
-      Y_Plus = 0.0; // to avoid warning
+      unsigned long counter = 0;
+      su2double diff = 1.0, Eddy_Visc = 1.0;
+      su2double U_Tau = sqrt(WallShearStress/Density_Wall);
+      su2double Y_Plus = 5.0;
 
       su2double Y_Plus_Start = Density_Wall * U_Tau * WallDistMod / Lam_Visc_Wall;
 
       /*--- Automatic switch off when y+ < 5 according to Nichols & Nelson (2004) ---*/
 
-      if (Y_Plus_Start < 5.0) {
-        cout << "skipping stress due to wall model" << endl;
-        continue;
+      if (Y_Plus_Start < Y_Plus) {
+        skipCounter++;
       }
-
-      while (fabs(diff) > tol) {
+      else while (fabs(diff) > tol) {
```

Comment:
I also made this logic the same as the incomp solver, it seemed that certain quantities would not be computed otherwise.
Hopefully this was the right thing to do and not the opposite.